english/marshmallow-attack/marsh-attack.md

```diff
@@ -0,0 +1,20 @@
+Noticing that the marshmallow wants to get the lighter
+
+you hold it out and pretend to look away. As the marsh hand 
+
+reaches for the lighter you quickly jab the fork into the 
+
+white fluffy hand. The marshmallow pulls back it hand and
+
+a giant gaping mouth appears and starts to scream.
+
+Do you
+
+[Light a fire?] (https://github.com/udacity/create-your-own-adventure/blob/master/english/light-fire/fire.md)
+
+[Dig a hole?] (https://github.com/udacity/create-your-own-adventure/blob/master/english/hands/hands.md)
+
+[Start eating?] (https://github.com/udacity/create-your-own-adventure/blob/master/english/eating-walls/eating-marshmallows.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.